push-images

```diff
@@ -26,25 +26,32 @@ while [ $# -gt 0 ]; do
     esac
 done
 
-push_image() {
-    local image="$1"
-    docker push "${image}:${IMAGE_TAG}"
-}
-
+pids=""
 for image in ${IMAGES}; do
     if [[ "$image" == *"build"* ]]; then
         continue
     fi
     echo "Will push ${image}:${IMAGE_TAG}"
-    push_image "${image}" &
+    docker push "${image}:${IMAGE_TAG}" &
+    pids+=" $!"
 
     if [ -z "$NO_DOCKER_HUB" ]; then
         # remove the quey prefix and push to docker hub
         docker_hub_image=${image#$QUAY_PREFIX}
         docker tag "${image}:${IMAGE_TAG}" "${docker_hub_image}:${IMAGE_TAG}"
         echo "Will push ${docker_hub_image}:${IMAGE_TAG}"
-        docker push "${docker_hub_image}:${IMAGE_TAG}"
+        docker push "${docker_hub_image}:${IMAGE_TAG}" &
+        pids+=" $!"
+    fi
+done
+
+EC=0
+for p in $pids; do
+    if ! wait $p; then
+        echo "Process $p failed with $?"
```

Comment:
I don't think we need this. The pid is going to be meaningless. And any error should already have been displayed.

So I'd simply exit on the first non-zero.